src/main/java/org/broadinstitute/hellbender/engine/spark/ShuffleJoinReadsWithRefBases.java

```diff
@@ -51,9 +51,10 @@
  * The reference bases paired with each read can be customized by passing in a reference window function
  * inside the {@link ReferenceDataflowSource} argument to {@link #addBases}. See {@link org.broadinstitute.hellbender.engine.dataflow.datasources.RefWindowFunctions} for examples.
  */
-public class JoinReadsWithRefBases {
+public class ShuffleJoinReadsWithRefBases {
     public static JavaPairRDD<GATKRead, ReferenceBases> addBases(final ReferenceDataflowSource referenceDataflowSource,
                                                                  final JavaRDD<GATKRead> reads) {
+        // TODO: reimpl this method by calling out to the more complex version?
```

Comment:
This is what I was asking above.  I can de-duplicate by first mapping to a `JavaPairRDD` where the value is always `null`.  This adds some overhead, but should be minimal.  Thoughts?